### DISCUSSION TITLE
HTTP methods in notebooks

### DISCUSSION THREAD

**Discussion by abhid** [marimo-team/marimo#580] [Ideas]
Is there an easy way to capture and act on common HTTP values like GET / POST params, Cookies, etc.?
Given that you're using Tornado, I'm guessing that there is a way to hook into variables exposed there.
I'm fairly new to the Python ecosystem but would love to contribute if there is interest.

**Comment by mscolnick** [comment]
Where do you want to listen to these events? In the marimo editor/python cells?

Currently we don't expose a way to listen to these events from the marimo editor. What are you trying to do specifically?
